test/karma/karma.conf.js

```diff
@@ -72,6 +72,7 @@ module.exports = function( config ) {
 
 		colors: !isTravis,
 
+		hostname: "bs-local.com",
```

Comment:
@sd5869 Could you set it to `bs-local.com` instead of `localhost` only if the following condition is truthy:
```js
process.env.BROWSER_STACK_USERNAME && process.env.BROWSER_STACK_ACCESS_KEY
```
? This is a pattern we use in this file down below.